src/main/resources/application.yml

```diff
@@ -100,6 +100,19 @@ features:
         update-prison-location: false
   apply-limited-access-markers: false
 
+new-nsi:
+  referral:
+    provider-code: CRS
+    staff-code: CRSUATU
+    team-code: CRSUAT
+    nsi-status: INPROG
+    referral-type-to-nsi-type: '{
+      "Accommodation": "CRS01",
+      "Education, Training and Employment (ETE)": "CRS02",
+      "Finance, Benefits and Debt (FBD)": "CRS03",
+      "Dependency and Recovery (D&R)": "CRS04"
```

Comment:
CRS05 - CRS07 are also in the reference data that's been set up for Interventions

Reply:
They haven't been included as there is no direct mapping from service categories in Interventions to CRS05-07. This is under review and the current thinking is service category will probably be replaced with Contract Title. However, this work will feature as part of another PR.